database/seeds/OldTeamsRecoverSeeder.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class OldTeamsRecoverSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $userTeams = DB::table('user_team')->select('user_id', 'team_id', 'role')->get();
+
+        foreach ($userTeams as $userTeam) {
+            $roleTypeId = '';
+            if ($userTeam->role === 'owner') {
+                $roleTypeId = 1;
+            }
+            else if ($userTeam->role === 'collaborator') {
+                $roleTypeId = 2;
+            }
+
```

Comment:
elseif should be written as a single keyword.
Secondly, elseif should start where the if{} block bracket ends.